File: app/src/main/java/org/michaelbel/moviemade/rest/api/service/REVIEWS.java
package org.michaelbel.moviemade.rest.api.service;

import org.michaelbel.moviemade.rest.model.v3.Review;

import io.reactivex.Observable;
import retrofit2.http.GET;
import retrofit2.http.Path;
import retrofit2.http.Query;

public interface REVIEWS {

    @GET("review/{review_id}")
    Observable<Review> getDetails(
        @Path("review_id") String id,
        @Query("api_key") String apiKey
    );
}

File: app/src/main/java/org/michaelbel/moviemade/data/service/REVIEWS.kt
package org.michaelbel.moviemade.data.service

import io.reactivex.Observable
import org.michaelbel.moviemade.data.entity.Review
import retrofit2.http.GET
import retrofit2.http.Path
import retrofit2.http.Query

interface REVIEWS {

    @GET("review/{review_id}?")
    fun getDetails(
        @Path("review_id") id: String,
        @Query("api_key") apiKey: String
    ): Observable<Review>
}